pkg/raft/rawnode_test.go

```diff
@@ -741,6 +741,7 @@ func TestRawNodeCommitPaginationAfterRestart(t *testing.T) {
 	})
 
 	rawNode, err := NewRawNode(cfg)
+	rawNode.raft.accTerm = 1
```

Comment:
This should not be needed when you make the `newRaft` func initialize `accTerm` to the last entry term.